client/src/pages/student/Registration.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import * as yup from 'yup';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+export default function Registration() {
+
+  function registration(data) {
+
+  }
+
+  const schema = yup.object().shape({
+    name: yup.string().matches(/^[A-Za-z ]*$/, 'Please enter valid name').required("Name field cannot be empty*"),
+    email: yup.string().required("Email is required*").matches(/^[\w-\\.]+@([\w-]+\.)+[\w-]{2,4}$/g, 'Email must be valid'),
```

Comment:
Use single quotes to represent string values.

Reply:
Added single quotes to represent string value.